gradle.properties

```diff
@@ -1,17 +1,13 @@
 #CHROME or FIREFOX
 browser=CHROME
 
-#features
-featureFolders =
```

Comment:
it has a purpose that I will need to teach you (all the class). 
Dont remove it.